### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kathedrall**
project structure, framework lib, elevation architecture and control panel

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@ services:
...
        XDEBUG_CONFIG: "remote_host=172.17.0.1 remote_enable=1" 
        DB_DRIVER_DEVELOPMENT: "mysql"
-       DB_HOST_DEVELOPMENT: "db"
-       DB_DATABASE_DEVELOPMENT: "dashboard"
-       DB_USERNAME_DEVELOPMENT: "root"
-       DB_PASSWORD_DEVELOPMENT: "root"
+       DB_HOST_DEVELOPMENT: "72.29.77.223"
+       DB_DATABASE_DEVELOPMENT: "givercardwebcom_voip"
+       DB_USERNAME_DEVELOPMENT: "givercardwebcom_voip"
+       DB_PASSWORD_DEVELOPMENT: "@VoxlinkDP"
```
Comment (vitormattos) [inline docker-compose.yml]: remove username and password and put in .env file

File: www/app/Controllers/BaseController.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+namespace App\Controllers;
+
+use Twig\Loader\FilesystemLoader;
+use Twig\Environment;
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use Monolog\Handler\FirePHPHandler;
+
+/**
+ * Controller basico, contem metodos comuns a todos os outros controllers da
```
Comment (vitormattos) [inline www/app/Controllers/BaseController.php]: English in all comments

File: www/app/Controllers/BaseController.php [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+     * @param string|array $key Chave de identificaçao da variavel. Pode ser uma
+     *                          string ou um array de valores.
+     * @param mixed $value Valor da variavel. Caso a variavel $key seja um array,
+     *                     ela deve ficar em branco.
+     * @return $this
+     */
+    public function setVar($key, $value = null)
+    {
+        if (is_array($key))
+        {
```
Comment (vitormattos) [inline www/app/Controllers/BaseController.php]: Invalid syntax, look PSR2

Comment (vitormattos) [reply www/app/Controllers/BaseController.php]: Suggest: use phpcs in continuous integration
see:
https://github.com/LyseonTech/ecommerce-data-colector/blob/master/composer.json#L47

File: www/app/Controllers/BaseController.php [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+     *                                      ou imprime diretamente.
+     * @return mixed
+     */
+    public function render(String $view) 
+    {
+        echo $this->view->render($view, $this->vars);
+    }
+    
+    /**
+     * Adiciona uma variavel para ser inserida no template.
```
Comment (vitormattos) [inline www/app/Controllers/BaseController.php]: Translate all comments

File: www/app/Models/Company.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
```
Comment (vitormattos) [inline www/app/Models/Company.php]: remove from all files

File: www/app/Views/login.html [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+<head>
+    <meta charset="UTF-8">
+    <link rel="shortcut icon" type="image/x-icon"
+        href="https://static.codepen.io/assets/favicon/favicon-aec34940fbc1a6e787974dcd360f2c6b63348d4b1f4e06c77743096d55480f33.ico" />
+    <link rel="mask-icon" type=""
+        href="https://static.codepen.io/assets/favicon/logo-pin-8f3771b1072e3c38bd662872f6b673a722f4b3ca2421637d5596661b4e2132cc.svg"
+        color="#111" />
+    <title>Template Login - VoxLink</title>
+    <style>
+        * {
```
Comment (vitormattos) [inline www/app/Views/login.html]: Move CSS to css file

File: www/app/Views/login.html [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="shortcut icon" type="image/x-icon"
+        href="https://static.codepen.io/assets/favicon/favicon-aec34940fbc1a6e787974dcd360f2c6b63348d4b1f4e06c77743096d55480f33.ico" />
+    <link rel="mask-icon" type=""
+        href="https://static.codepen.io/assets/favicon/logo-pin-8f3771b1072e3c38bd662872f6b673a722f4b3ca2421637d5596661b4e2132cc.svg"
+        color="#111" />
+    <title>Template Login - VoxLink</title>
```
Comment (vitormattos) [inline www/app/Views/login.html]: put enterprise name in .env vile

File: www/app/Views/login.html [hunk 8]
```diff
@@ -0,0 +1,399 @@
...
+        </div>
+        <div class="group">
+            <input type="email"><span class="highlight"></span><span class="bar"></span>
+            <label>Senha</label>
+        </div>
+        <button type="button" class="button buttonBlue">Entrar
+            <div class="ripples buttonRipples"><span class="ripplesCircle"></span></div>
+        </button>
+    </form>
+    <footer><a href="https://www.dprimeira.com.br/" target="_blank"><img
```
Comment (vitormattos) [inline www/app/Views/login.html]: Replace this footer. The company is LyseonTech.

File: www/app/Views/login.html [hunk 9]
```diff
@@ -0,0 +1,399 @@
...
+                src="https://www.polymer-project.org/images/logos/p-logo.svg"></a>
+        <p>Projeto <a href="https://www.dprimeira.com.br/" target="_blank">xxxx</a></p>
+    </footer>
+    <script
+        src="https://static.codepen.io/assets/common/stopExecutionOnTimeout-de7e2ef6bfefd24b79a3f68b414b87b8db5b08439cac3f1012092b2290c719cd.js"></script>
+    <script src='//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
```
Comment (vitormattos) [inline www/app/Views/login.html]: Why old version of jQuery?

File: www/app/Views/login.html [hunk 10]
```diff
@@ -0,0 +1,399 @@
...
+                $this.addClass('is-active');
+
+            });
+
+            $ripples.on('animationend webkitAnimationEnd mozAnimationEnd oanimationend MSAnimationEnd', function (e) {
+                $(this).removeClass('is-active');
+            });
+
+        });
+      //# sourceURL=pen.js
```
Comment (vitormattos) [inline www/app/Views/login.html]: Put the js content in js file

File: www/app/Controllers/BaseController.php (hunk 3 above)
Comment (vitormattos) [reply www/app/Controllers/BaseController.php]: Look issue #10

File: www/app/Views/login.html (hunk 6 above)
Comment (kathedrall) [reply www/app/Views/login.html]: created css file

File: www/app/Controllers/BaseController.php (hunk 2 above)
Comment (kathedrall) [reply www/app/Controllers/BaseController.php]: Perfect. I'll be commenting all routines on delivery better

File: www/app/Controllers/BaseController.php (hunk 4 above)
Comment (kathedrall) [reply www/app/Controllers/BaseController.php]: Perfect. I'll be commenting all routines on delivery better

File: www/app/Views/login.html (hunk 8 above)
Comment (kathedrall) [reply www/app/Views/login.html]: ok

File: www/app/Models/Company.php (hunk 5 above)
Comment (kathedrall) [reply www/app/Models/Company.php]: ok

File: www/app/Views/login.html (hunk 10 above)
Comment (kathedrall) [reply www/app/Views/login.html]: created

File: www/app/Views/login.html (hunk 9 above)
Comment (kathedrall) [reply www/app/Views/login.html]: I removed this tag by accessing the old jquery version

File: www/app/Views/login.html (hunk 7 above)
Comment (kathedrall) [reply www/app/Views/login.html]: created

File: docker-compose.yml (hunk 1 above)
Comment (kathedrall) [reply docker-compose.yml]: created

**Pull request closed without merging**